.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
-  - 5.4
```

Comment:
Not critical, but for 1.0.x-dev which should keep eZ Publish 5.4 compat which support PHP 5.4, PHP 5.4 testing should remain, kernel 6.0.x is available publicly for testing on php 5.4 for instance.